example/pubspec.yaml

```diff
@@ -36,10 +36,7 @@ dependency_overrides:
     git:
       url: git://github.com/fleaflet/flutter_map.git
       ref: issues/829-nullsafety
-  flutter_map_marker_popup:
```

Comment:
if removed from dev_overrides, does a nullsafe version need to be added to dependencies?

Reply:
Hmm I didn't try but maybe I should not have removed this one. Maybe add it back and then when flutter_map finally releases null safe and then I release null safe flutter_map_marker_popup we can remove these anyway.

Reply:
Done now that it's moved to the new version.